plugins/search/src/components/SearchType/SearchType.Accordion.tsx

```diff
@@ -147,9 +147,7 @@ export const SearchTypeAccordion = (props: SearchTypeAccordionProps) => {
 
   return (
     <Box>
-      <Typography variant="body2" component="h2">
-        {name}
-      </Typography>
+      <Text variant="body">{name}</Text>
```

Comment:
The removal of `h2` here seems like it might hurt accessibility, is there an equivalent option available? Or maybe it was never necessary to begin with.

Reply:
Yes you can use the render prop to render it as h2.